jenkins/run-pytests

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+prog="$(basename "${0}")"
+progdir="$(realpath -e $(dirname "${0}")/..)"
+
+source $(dirname "${0}")/python-setup.sh
+
+printf -- "black --check ...\n"
+black --check .
+black_sts=${?}
+if [[ ${black_sts} -ne 0 ]]; then
+    printf -- "FAILED: black --check .\n" >&2
+fi
+
+printf -- "\nflake8 --verbose ...\n"
+flake8 --verbose .
+flake_sts=${?}
+if [[ ${flake_sts} -ne 0 ]]; then
+    printf -- "FAILED: flake8 --verbose .\n" >&2
+fi
+printf -- "\n"
+
+pytest {posargs} ./server/lib/pbench/test/unit
```

Comment:
Instead of modifying 4 files, and repeating infrastructure code in `run-pytests`, why not just add these four lines to `run-unittests` after the `black` and `flake8` invocations?  That way you only modify one file for this commit.